src/Native/System.Native/pal_networking.h

```diff
@@ -117,6 +117,23 @@ struct LingerOption
     int32_t Seconds; // Number of seconds to linger for
 };
 
+struct IOVector
+{
+    uint8_t* Base;
+    uintptr_t Count;
+};
+
+struct MessageHeader
+{
+    uint8_t* SocketAddress;
```

Comment:
Practically speaking, you're correct--any reasonable system I'm aware of uses 8 bits for a `char`. As mentioned, though, it's better to be explicit.